### ISSUE TITLE
Demo: grasp an object on a table with minimal pressure

### ISSUE THREAD

**Issue by lornat75** [tacman-fp7/tactile-control#13]
Show that the pressure control allows applying minimal pressure (assuming it is enough for lifting), and grasp objects of different size (diameters).

The position of the object could be fixed, preprogram the arm to reach the correct position then close the fingers and lift the object.

**Comment by lornat75** [comment]
@pattacini @massimoregoli

**Comment by lornat75** [comment]
any updates on this :) ?

**Comment by pattacini** [comment]
I think @massimoregoli needs first to accomplish #12 at least...
Controlling two phalanxes at once (see #14) could help out, but to start with we can keep this aside.

**Comment by massimoregoli** [comment]
At the moment I'm focusing on #12, that anyway can be considered part of the demo grasping task, since both the thumb and the middle finger should be involved

**Comment by massimoregoli** [comment]
Exactly :-) we posted at the same time

**Comment by pattacini** [comment]
I'd consider the index finger too for a human-like grasp. This shouldn't make any big difference though.

**Comment by massimoregoli** [comment]
it shouldn't be a problem when working with the black and blue icub. For the purple one it could be a bit more complicated, due to the different positioning of the thumb.

**Comment by pattacini** [comment]
Yes, but there'll be the object in the middle and won't be so small :smirk:

**Comment by massimoregoli** [comment]
Yes I agree it depends on the dimension of the object to grasp, anyway i got some troubles trying to find a grasp position using the paper cup.

**Comment by massimoregoli** [comment]
Anyway of course I will develop the software so that we can manage whatever joint we want
